bench.sh

```diff
@@ -265,6 +265,18 @@ function tests_run_add_remove { # <test> <add?>
   fi
 }
 
+function process_external { # file
```

Comment:
"processing" doesn't say much about what it's doing, maybe rename it "read_external_allocator_from_file" ?